bevy_matchbox/src/socket.rs

```diff
@@ -77,8 +95,8 @@ impl<C: BuildablePlurality> From<WebRtcSocketBuilder<C>> for MatchboxSocket<C> {
 impl<C: BuildablePlurality> From<(WebRtcSocket<C>, MessageLoopFuture)> for MatchboxSocket<C> {
     fn from((socket, message_loop_fut): (WebRtcSocket<C>, MessageLoopFuture)) -> Self {
         let task_pool = IoTaskPool::get();
-        task_pool.spawn(message_loop_fut).detach();
```

Comment:
I thought detach was necessary to actually poll it in the background by bevy. Are we sure removing this is OK?

Reply:
Bevy tasks are eager, detach is simply a way of dropping the handle without cancelling the task